core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -454,13 +472,24 @@ private[chisel3] object Converter {
       )
   }
 
+  def rec(decl: GroupDecl): fir.GroupDeclare = {
+    val convention = decl.convention match {
+      case GroupConvention.Bind => fir.GroupConvention.Bind
+    }
+    decl match {
+      case leaf if leaf.children.isEmpty => fir.GroupDeclare(convert(leaf.sourceInfo), leaf.name, convention, Seq.empty)
+      case node                          => fir.GroupDeclare(convert(node.sourceInfo), node.name, convention, node.children.map(rec))
+    }
```

Comment:
```suggestion
    fir.GroupDeclare(convert(decl.sourceInfo), decl.name, convention, decl.children.map(rec))
```
Unless I'm missing something, these are identical and mapping on empty children will not recurse

Reply:
Good catch. The `Builder` has the same problem. Will fix that there, too.